py/release.py

```diff
@@ -19,8 +22,11 @@
 from py import build_and_push_image
 from py import util
 
-REPO_ORG = "tensorflow"
-REPO_NAME = "k8s"
+# Repo org and name can be set via environment variables when running
+# on PROW. But we choose sensible defaults so that we can run locally without
+# setting defaults.
+REPO_ORG = os.getenv("REPO_OWNER", "tensorflow")
```

Comment:
Does it make sense to make this change here as well for consistency? https://github.com/tensorflow/k8s/blob/master/py/util.py#L26

Reply:
Done.